documentation/modules/auxiliary/scanner/nfs/nfsmount.md

```diff
@@ -1,65 +1,102 @@
 ## Vulnerable Application
 
-NFS is very common, and this scanner searches for a mis-configuration, not a vulnerable software version.  Installation instructions for NFS can be found for every operating system.
-The [Ubuntu 14.04](https://help.ubuntu.com/14.04/serverguide/network-file-system.html) instructions can be used as an example for installing and configuring NFS.  The
+NFS is very common, and this scanner searches for a mis-configuration, not a vulnerable software version.
+Installation instructions for NFS can be found for every operating system.
+The [Ubuntu 14.04](https://help.ubuntu.com/14.04/serverguide/network-file-system.html)
```

Comment:
Ubuntu 14.04 was released 8 years ago and the link is a 404.

Here's a modern alternative:

```suggestion
The [Ubuntu 14.04](https://ubuntu.com/server/docs/service-nfs)
```